### PR TITLE
CircleCI setup

### PR DESCRIPTION
**Pull request by stevenjack**
### Problem

We want to build the library.

### Solution

Adds circleCI deps and versioning `Makefile`.

### Notes

- **Remember** to tag your merge commit after merging 👍

### CODE REVIEW

**Comment by stevenjack** [comment]
[[GPN] Adds circleCI setup](https://trello.com/c/PSjuJ017/897-gpn-adds-circleci-setup)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+VERSION = $(shell cat ./VERSION)
+
+check-version:
+	git fetch && (! git rev-list ${VERSION})
+
+push-tag:
+	git checkout ${BRANCH} \
+	&& git pull origin ${BRANCH} \
+	&& git tag ${VERSION} \
+	&& git push origin ${BRANCH} --tags
```
Comment (revett) [inline Makefile]: No need for `&&`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+BRANCH = "master"
+VERSION = $(shell cat ./VERSION)
+
+check-version:
+	git fetch && (! git rev-list ${VERSION})
```
Comment (revett) [inline Makefile]: Chuck these on two lines and remove the `&&`

File: glide.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+package: github.com/vidsy/go-paypal-nvp
+import: []
```
Comment (revett) [inline glide.yaml:2]: Change to `testImport`

**Review by revett — APPROVED** [review]
Few small comments 👍

**Pull request merged**